build.sbt

```diff
@@ -113,13 +100,13 @@ artifact in (Compile, assembly) := {
 addArtifact(artifact in (Compile, assembly), assembly)
 
 val resetSample = TaskKey[Unit]("resetSample", "Reset sample module")
-val scalaFrameworks = List("akkaHttp", "endpoints", "http4s")
-val javaFrameworks = List("dropwizard")
+val scalaFrameworks = exampleFrameworkSuites("scala").map(_._2)
+val javaFrameworks = exampleFrameworkSuites("java").map(_._2)
 
 resetSample := {
   import scala.sys.process._
   (List("sample") ++ (scalaFrameworks ++ javaFrameworks).map(x => s"sample-${x}"))
-    .foreach(sampleName => s"git clean -fdx modules/${sampleName}/src modules/${sampleName}/target" !)
+    .foreach(sampleName => s"rm -rf modules/${sampleName}/target/generated" !)
```

Comment:
Still would prefer to stick to `git clean` here, as it will refuse to eat all your data if things go wrong. Operating with the bounds of the repo as the limit to potential damage is 💯

Reply:
Yeah, good call.